### PR TITLE
Don't mix kickstart commands for installation methods

### PR DESCRIPTION
**Pull request by poncovka**
Make sure that the kickstart command `method` is used only
for the RPM sources, so we can later move the RPM sources
to the DBus module.

### CODE REVIEW

File: pyanaconda/payload/live/payload_liveos.py [hunk 1]
```diff
@@ -41,20 +42,36 @@ def type(self):
...
+    def _get_live_os_image():
+        """Detect live os image on the system.
+
+        :return: a path to the image
+        """
+        payloads_proxy = PAYLOADS.get_proxy()
+        source_path = payloads_proxy.CreateSource(SOURCE_TYPE_LIVE_OS_IMAGE)
+
+        source_proxy = PAYLOADS.get_proxy(source_path)
+        return source_proxy.DetectLiveOSImage()
```
Comment (jkonecny12) [inline pyanaconda/payload/live/payload_liveos.py:55]: This should be handled better way I think. It's fine for now but I don't think we should create source to be able to detect if there is a source. This should be ideally part of the payload?

Please add TODO here.

Comment (poncovka) [reply pyanaconda/payload/live/payload_liveos.py]: Added a comment.

**Comment by poncovka** [comment]
Rebased.

**Comment by poncovka** [comment]
Tested.

**Pull request merged**